File: two_d _array/Mul_D.java
public class Mul_D
{
    public static void main(String args[])
    {
        int arr[][]={{2,1},{4,2},{5,2}};
        System.out.println("arr="+arr);

        System.out.println("Array elements are=");
        for(int i=0;i<arr.length;i++)
        {
            for(int j=0;j<arr[0].length;j++)
            {
                System.out.print(arr[i][j]+" ");
            }
            System.out.println("");
        }
    }
}

File: two_d _array/Mul_D.py
# # arr=[[1,2],[3,4],[5,6]]
# # print(arr)
# print("Array elements are:")
print("Array elements are")
arr=[]
row=3
clmn=3
for i in range(row):
    temp=[]
    for j in range(clmn):
        ele=int(input("Enter the array ele"))
        temp.append(ele)
    arr.append(temp)

for i in range(len(arr)):
    for j in range(len(arr[0])):
        print(arr[i][j],end=" ")
    print()
 

